examples/event_handling/pipong.py

```diff
@@ -1,7 +1,10 @@
-# A matplotlib based game of Pong illustrating one way to write interactive
-# animation which are easily ported to multiple backends
-# pipong.py was written by Paul Ivanov <http://pirsquared.org>
+"""
+====
+PONG
+====
 
+A matplotlib based game of Pong illustrating one way to write interactive animation which are easily ported to multiple backends pipong.py was written by Paul Ivanov <http://pirsquared.org>
```

Comment:
matplotlib -> Matplotlib
animation -> animations
Add period at end of sentence.

Also, please wrap this line at 79 characters.